content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -427,7 +427,44 @@ For a TCP probe, the kubelet makes the probe connection at the node, not in the
 means that you can not use a service name in the `host` parameter since the kubelet is unable
 to resolve it.
 
+### Setting TerminationGracePeriodOverride
```

Comment:
```suggestion
### Per-probe terminationGracePeriod
```
I'm not sure what a TerminationGracePeriodOverride is - an implementation detail rather than part of the API?

Reply:
I just goofed up and wrote it wrong :zany_face: